ios/RNTPTDocumentView.m

```diff
@@ -1628,6 +1628,27 @@ - (NSDictionary *)getField:(NSString *)fieldName
     return [[fieldMap allKeys] count] == 0 ? nil : fieldMap;
 }
 
+- (NSMutableDictionary *)getFieldWithHasAppearance:(PTAnnot *)annot
```

Comment:
Looks good I think. Just missed changing the return type to the immutable version.

```suggestion
- (NSDictionary *)getFieldWithHasAppearance:(PTAnnot *)annot
```